### ISSUE TITLE
[Kill a Skeleton with Bees]

### ISSUE THREAD

**Issue by Ezneh** [Joshimuz/mcbingo#81]
**Kill a Skeleton with Bees**
The goal is to make a Skeleton die while fighting with Bees. Meaning that a Bee should do the "last hit" on the Skeleton.

**Difficulty level**
Most likely Hard

**Anti Synergies and Catalysts/Reactants**
Pacifist might be an anti-synergy?

**Why this Goal is good**
Gives another role to the Bees, and can replace the "Kill a Skeleton with it's own arrow" until it's fixed.

*Is it cool or funny? Like: "Kill yourself with your own arrow"
It can be funny to watch a Skeleton being killed by a Bee.

**Comment by Joshimuz** [comment]
How do you do this? Get a Skeleton to hit a Bee with it's arrow?

**Comment by Ezneh** [comment]
Yes basically, then the Bee fights back.

**Comment by Joshimuz** [comment]
I tried it in Creative mode and it seems way too hard, Skeletons can't get Poisoned so getting a Skeleton to the correct health would be quite a challenge, not sure it fits either Hard or Very Hard

**Comment by Joshimuz** [comment]
Maybe just "Make a Bee sting a Skeleton" or "sting another mob" or something, but that's not super interesting

**Comment by Ezneh** [comment]
> 
> 
> Maybe just "Make a Bee sting a Skeleton" or "sting another mob" or something, since the actual "idea" of the goal is just to make Bees angry at something other than you, not necessarily killing something, but that's not super interesting. Would also change it from a Hard to Medium or even Easy, but then it seems way too "fiddily" to do in either of those.

Yeah that's true. It is less interesting if it's just making them angry at another entity.

**Comment by Joshimuz** [comment]
Reviewing new goals for the new scoring system, 4 years ago Josh is smart in suggesting the goal be "Make a Bee sting another mob" so I'll try to add that now
